autopts/ptsprojects/zephyr/sm_wid.py

```diff
@@ -36,3 +37,19 @@ def hdl_wid_143(desc):
     btp.gap_read_ctrl_info()
 
     return True
+
+
+def hdl_wid_175(desc):
+    """
+    Please confirm that IUT reports the Confirm Value Failed to the Upper Tester
+    """
+
+    stack = get_stack()
+    _, _, reason = stack.gap.gap_wait_for_pairing_fail(10)
+
+    # BT_SMP_ERR_CONFIRM_FAILED(0x04) is converted to BT_SECURITY_ERR_AUTH_FAIL(0x01)
+    # in zephyr host.
+    if reason == 0x01:
```

Comment:
Might be a good idea to define this value instead of "magic" 0x01 - after that the comment might be unnecessary, as it becomes self-explanatory; then again, having a well commented code is always good - so leaving it is good too :).

Reply:
If this is a Zephyr workaround, then shouldn't this go in into https://github.com/auto-pts/auto-pts/blob/master/autopts/ptsprojects/zephyr/sm_wid.py instead of the generic file? 

Please also create a Zephyr GH issue or PR that we can refer to here if you intent on this being a temporary fix that depends on a change in Zephyr.

Reply:
Somehow I misread the path - My bad :D Thanks for creating the issue